### PR TITLE
FRC: Piece Gateway

### PR DESCRIPTION
**Pull request by willscott**
This FRC specifies the HTTP semantics for piece retrieval.

[rendered version](https://github.com/willscott/FIPs/blob/3c073e89d5c2df1b24742af748c4fbacb4e06846/FRCs/piece-gateway.md)

### CODE REVIEW

**Comment by willscott** [comment]
Discussion: https://github.com/filecoin-project/FIPs/discussions/711

File: FRCs/piece-gateway.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
+| fip | title | author | discussions-to | status | type | created |
+| --- | --- | --- | --- | --- | --- | --- |
+| TBD | Piece Retrieval Gateway | Will Scott (@willscott), Dirk McCormick (@dirkmc) | TBD | Draft | FRC | 2023-05-19 |
```
Comment (anorth) [inline FRCs/piece-gateway.md:3]: Please use the template format for these headers so they are amenable to any future automation (e.g. creating the table in README)

**Review by anorth — APPROVED** [review]
Editorial rubber stamp. I have not done a deep technical review of this FRC.

**Review by kaitlin-beegle — APPROVED** [review]

**Comment by kaitlin-beegle** [comment]
I have reviewed for content.  I cannot perform technical diligence on this document, but given last week's presentation to Core Devs, and the fact that it's an FRC, I'm going to merge.

**Pull request merged**